packages/components/src/Components/ConditionalFilter/ConditionalFilter.js

```diff
@@ -32,7 +32,7 @@ class ConditionalFilter extends Component {
             items[0]
         );
         const onChangeCallback = onChange || this.onChange;
-        const ActiveComponent = activeItem && typeMapper(activeItem.type);
+        const ActiveComponent = activeItem && (typeMapper[activeItem.type] || typeMapper.text);
```

Comment:
oh oh oh  awesome, so we can either explicitly declare text type or just not n its the default 👍